client/src/components/profile/AuthenticationCard.vue

```diff
@@ -0,0 +1,60 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+<template>
+  <div
+    class="authentication-card"
+    v-show="showCard"
+  >
+    <img
+      :src="imageSrc"
+      :alt="imageAlt"
+      class="authentication-card__image"
+    />
+    <ion-text class="authentication-card__text subtitles-normal">{{ $msTranslate(methodName) }}</ion-text>
+    <ion-icon
+      :icon="checkmarkCircle"
+      class="authentication-card__icon"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { IonText, IonIcon } from '@ionic/vue';
+import { checkmarkCircle } from 'ionicons/icons';
+import { Translatable } from 'megashark-lib';
+
+defineProps<{
+  imageSrc: string;
+  imageAlt: string;
+  methodName: Translatable;
+  showCard?: boolean;
```

Comment:
What's the point of this prop? If you display nothing when set, what's the difference between
```vue
<authentication-card
  :show-card="false"
/>
```
and
```vue
<authentication-card
  v-show="false"
/>
```